sdk/textanalytics/Azure.AI.TextAnalytics/README.md

```diff
@@ -14,7 +14,7 @@ Azure Cognitive Services Text Analytics is a cloud service that provides advance
 Install the Azure Text Analytics client library for .NET with [NuGet][nuget]:
 
 ```PowerShell
-dotnet add package Azure.AI.TextAnalytics -Version 1.0.0-preview.3
+dotnet add package Azure.AI.TextAnalytics --version 1.0.0-preview.4
```

Comment:
Just curious, why is version’s casing changing

Reply:
In a previous PR (https://github.com/Azure/azure-sdk-for-net/pull/10975) we changed from `Install-Package` to `dotnet add package` and I didn't update the version parameter.
@JoshLove-msft pointed out :)